### DISCUSSION TITLE
Unable to create any new codespaces.

### DISCUSSION THREAD

**Discussion by kingces95** [community/community#35963] [Codespaces] [answered]
When I try to crate new codespace a get a generic "something went wrong" 500 message. Has the beta expired? Is there somewhere I can go to renew. My github is kingces95.

Thanks

**Comment by cty7** [comment]
Same thing is happening to me rn

**Comment by kingces95** [comment]
When I try from VSCode to create new codespace I get:

Connect to Codespace: Get codespace request failed with status 500. Correlation ID: C5E5:594D:2FB770:30E521:63474A30

**Comment by kingces95** [comment]
This problem resolved but is now re-occuring. I was using the python debugger when this happened but don't know if that is the cause.

**Comment by thomas-sickert** [comment, marked as answer]
Sorry you were having trouble connecting! Is it resolved now?

It looks like this was related to a [Codespaces incident](https://www.githubstatus.com/incidents/7hfyz8mjg663). To check the status of the Codespaces service, you can visit the [GitHub status page](https://www.githubstatus.com/) or check the [@githubstatus twitter account](https://twitter.com/githubstatus).
